test/common/integration/quic_test_server_interface.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include "test/common/integration/quic_test_server.h"
+
+// NOLINT(namespace-envoy)
+
+#ifdef __cplusplus
+extern "C" { // functions
+#endif
+
+void start_server();
```

Comment:
nit: Should we have brief comments for these methods?

Reply:
Done